dcpy/utils/postgres.py

```diff
@@ -27,9 +27,14 @@ def generate_engine_uri(
     return server_url + "/" + database + options
 
 
-def execute_file_via_shell(engine_uri: str, path: Path) -> None:
+def execute_file_via_shell(
+    engine_uri: str, path: Path, psql_vars: list[str] | None = None
```

Comment:
I'll see where this is called before passing full judgement but maybe this should either be a dict, or just **kwargs?